cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -289,11 +328,12 @@ def prepare_df(
     )
     df = df.withColumn("Promo2Weeks", (df.Promo2Days / 7).cast(T.IntegerType()))
 
-    # dheck that we did not lose any rows through inner joins
+    # ensure that no row was lost through inner joins
     assert num_rows == df.count(), "lost rows in joins"
     return df
 
-
+# %%
+# We define a function that builds a dictionary of sorted categorical variables.
```

Comment:
We build a dictionary...
"define a function" is everywhere. Let's rephrase the sentences.